templates/zerver/api/writing-bots.md

```diff
@@ -30,14 +30,16 @@ On this page you'll find:
 
 2. `cd python-zulip-api` - navigate into your cloned repository.
 
-3. `./tools/provision` - install all requirements in a Python virtualenv.
+3. `python3 ./tools/provision` - install all requirements in a Python virtualenv.
 
-4. Run the `source <activation/path>` command printed in the previous
-   step to activate the virtualenv.
+4. `source /.../python-zulip-api/.../activate` - if the `python3 ./tools/provision`
+   command succeeds, it will end with printing a `source` command. You can run this
+   command to activate the virtual environment.
```

Comment:
This feels like too much text for what we're explaining.

Reply:
I'll shorten it. :)